models/lecture.js

```diff
@@ -0,0 +1,16 @@
+const uuid = require('uuid/v4');
+'use strict';
+// var models = require('../models');
+module.exports = (sequelize, DataTypes) => {
+    var Lecture = sequelize.define('Lecture', {
+        id: { type: DataTypes.UUIDV4, primaryKey: true, defaultValue: uuid() },
+        date: DataTypes.TEXT,
```

Comment:
Change to DATEONLY

http://docs.sequelizejs.com/variable/index.html

Reply:
done